src/applications/edu-benefits/10203/containers/ConfirmEligibilityView.jsx

```diff
@@ -107,125 +103,71 @@ export class ConfirmEligibilityView extends React.Component {
+                <div className="usa-alert-body">
+                  <strong>Your remaining education benefits</strong>
+                  <div className="usa-alert-text">
+                    <p>
+                      Our entitlement system shows that you have more than 6
+                      months of education benefits remaining. You should apply
+                      when you have less than 6 months of entitlement left.
+                    </p>
+                    <p>
+                      Months you have left to use:{' '}
+                      <strong>
+                        {this.props.remainingEntitlement.months} months,{' '}
+                        {this.props.remainingEntitlement.days} days{' '}
+                      </strong>
+                    </p>
                   </div>
                 </div>
-                <br />
               </div>
-            )}
-          <span>
-            If your situation changes in the future and you meet all of the
-            criteria, you may return to apply for the Rogers STEM Scholarship.
-          </span>
-          <div className="vads-u-padding-top--2">
-            <a
-              className={'usa-button-primary wizard-button va-button-primary'}
-              href="/education/about-gi-bill-benefits/"
-            >
-              Explore other education benefits
-            </a>
-          </div>
-        </div>
-
-        <div>
-          <fieldset className="schemaform-field-template schemaform-first-field">
-            <legend className={legendClassName}>
-              {questionText}
-              <span className="schemaform-required-span">(*Required)</span>
-            </legend>
-            {this.renderErrorMessage()}
-            <input
-              type="radio"
-              checked={confirmEligibility === false}
-              id={`${id}No`}
-              name={`${id}`}
-              value="N"
-              onChange={() =>
-                this.onChange({ 'view:confirmEligibility': false })
-              }
-              onClick={() => captureEvents.ineligibilityStillApply(false)}
-            />
-            <label htmlFor={`${id}No`}>No</label>
-            <input
-              type="radio"
-              checked={confirmEligibility === true}
-              id={`${id}Yes`}
-              name={`${id}`}
-              value="Y"
-              onChange={() =>
-                this.onChange({ 'view:confirmEligibility': true })
-              }
-              onClick={() => {
-                captureEvents.ineligibilityStillApply(true);
-              }}
-            />
-            <label htmlFor={`${id}Yes`}>Yes</label>
-          </fieldset>
-        </div>
+            </div>
+          )}
       </div>
     );
   };
 
   render() {
     return (
       <div>
+        <div className="vads-u-padding-bottom--2">
+          <h4>Rogers STEM Scholarship eligibility summary</h4>
```

Comment:
this doesn't go here, this should be the `title` property of the page in `src/applications/edu-benefits/10203/config/chapters.js`

Reply:
The config should look like `src/applications/edu-benefits/10203/config/chapters.js`
```
      confirmEligibility: {
        title: 'Rogers STEM Scholarship eligibility summary',
        path: 'benefits/confirm-eligibility',
        depends: form => displayConfirmEligibility(form),
        pageClass: 'vads-u-max-width--100 vads-u-vads-u-width--full',
        uiSchema: confirmEligibility.uiSchema,
        schema: confirmEligibility.schema,
        editModeOnReviewPage: false,
      },
```

Setting `title` will correctly put the title on the page and on the review page.
Setting `editModeOnReviewPage` should disable the edit button from appearing.

Reply:
This should be the config for the page, notice `hideHeaderRow` property this removes the edit button
```
      confirmEligibility: {
        title: 'Rogers STEM Scholarship eligibility summary',
        path: 'benefits/confirm-eligibility',
        depends: form => displayConfirmEligibility(form),
        pageClass: 'vads-u-max-width--100 vads-u-vads-u-width--full',
        uiSchema: confirmEligibility.uiSchema,
        schema: confirmEligibility.schema,
        hideHeaderRow: true,
      },
```